### ISSUE TITLE
PlanToTSR fails with attribute error: self._TsrSampler

### ISSUE THREAD

**Issue by jeking04** [personalrobotics/or_trajopt#2]
Currently the PlanToTSR function fails because there is no self._TsrSampler defined. I think there may have been a partial merge somewhere. Or do we need to define this method when we create the planner?

**Comment by rachelholladay** [comment]
It was defined in constraints, so it should have been constraints._TsrSampler. There were a couple of other things that I just redid not realizing I was redoing the work in the branch bugfix/tsr. @psigen was there a reason that branch hasn't been merged in yet?

**Comment by psigen** [comment]
Yes, it's because after implementing the projection the optimization failed all the time in local collision minima, which is why the PlanToTSR function isn't a `@PlanningMethod` in master.  @siddhss5 was working on a better way of projecting using nlopt.

@rachelholladay: you're welcome to try to continue the branch bugfix/tsr, I'm planning to come back to it at some point in the next month but I don't know when.  It's possible there was a bug in my projection implementation.

**Comment by rachelholladay** [comment]
Coolio. I already opened a separate issue, #5, for @siddhss5 to fix it, since he said he would :). So between that and the fixes in bugfix/tsr hopefully PlanToTSR will be up soon.
